samples/crypto/aes_ctr/README.rst

```diff
@@ -0,0 +1,62 @@
+.. _crypto_aes_ctr:
+
+Crypto: AES CTR
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The AES sample shows how to perform AES encryption and decryption operations using the CTR mode without padding using an 128 bit AES key.
+
+It does the following things:
+
+* Encrypts a sample plaintext:
+   * Initalizes the PSA apis
```

Comment:
```suggestion
   * The Platform Security Architecture (PSA) API is initialized.
```
Spell out acronyms when used for the first time, we cannot assume customers are familiar with all of them.